packages/website/src/theme/CodeBlock/Content/String.tsx

```diff
@@ -90,6 +92,7 @@ export default function CodeBlockString({
               <code
                 className={clsx(
                   styles.codeBlockLines,
+                  needsMorePadding && styles.codeBlockLinesMorePadding,
```

Comment:
Ahh, this is still adding more padding even when there's not a playground button. Example from https://deploy-preview-10019--typescript-eslint.netlify.app/getting-started/#step-1-installation:

![Screenshot of the linked 'Step 1: Installation' section. The 'npm install' codeblock has too extra vertical padding, but no 'Show in Playground' button.](https://github.com/user-attachments/assets/3e44c075-d568-466a-9761-e010473a96a4)

I think we should only add the padding for codeblocks that have the playground button.

Reply:
I missed this part again. Thanks for reviewing it